components/admission-webhook/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV GO111MODULE=on
 RUN go build  -o webhook .
```

Comment:
I'm assuming that this part is what prevents us from using the static image.

`CGO_ENABLED=0 GOOS=linux go build -o webook -ldflags "-w" -a .`
Could you please try this instead?
Without `CGO` we should be able to build with static

Reply:
Using base-debian10

Reply:
@swiftdiaries where is that line `CGO_ENABLED=0 GOOS=linux go build -o webook -ldflags "-w" -a .` 
it's not in the Makefile

Reply:
I think @swiftdiaries is suggesting changing the existing go-build line to what he provided. I think setting CGO_ENABLED=0 removes a dependency on some c stuff in the compiled binary which might allow the use of static instead of base.

Reply:
@swiftdiaries 
I have the below in the Dockerfile;
```
# Build
#RUN go build  -o webhook .
RUN CGO_ENABLED=0 GOOS=linux go build -o webook -ldflags "-w" -a .
# Copy the controller-manager into a distroless image
FROM gcr.io/distroless/base-debian10:latest
WORKDIR /
COPY --from=builder /go/src/github.com/kubeflow/kubeflow/components/admission-webhook/webhook .
```

But I'm getting the below errors;
```
COPY failed: stat /var/lib/docker/overlay2/a11ca9ddcef163c8b24f630971afd113cf3f45adae11a9fd0cb81ef5a862e6e5/merged/go/src/github.com/kubeflow/kubeflow/components/admission-webhook/webhook: no such file or directory
```

Reply:
@jeff in the line:
```
RUN CGO_ENABLED=0 GOOS=linux go build -o webook -ldflags "-w" -a .
```
Webhook is mis-spelt.